### DISCUSSION TITLE
What is your favorite YouTube channel?

### DISCUSSION THREAD

**Discussion by kozmr** [NAIST-SE/AttractingContributors#1113] [2024 Attraction Contributions]
好きなYouTubeチャンネル教えてください．なんでもOK．
What is your favorite YouTube channel? Any field is welcome.
I like "Stuff Made Here".
https://github.com/kozmr/AttractingContributors-YouTube

**Comment by gill876** [comment]
[Caitlin Doughty](https://www.youtube.com/@AskAMortician) is mine! Highly recommend if you're into documentaries and death lol.

Please consider making a contribution to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by 3207-Rhims** [comment]
#1161 can you make one PR, I made a PR for you.

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Reply by anamhamza786** [reply]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git

**Comment by r-nishioka** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[Where would you like to trip in Japan?
](https://github.com/r-nishioka/AttractingContributors-Trip.git)

**Reply by anamhamza786** [reply]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git

**Comment by kozmr** [comment]
Thank you for pull requests.
Please 2 pull requests.

**Comment by Marsall-k** [comment]
I have just made a PR to your Repo.
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Reply by anamhamza786** [reply]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
